Remotion/Web/Test.Shared/IFrameSupport/MainForm.aspx.cs

```diff
@@ -16,13 +16,18 @@
 // 
 using System;
 using System.Web;
+using Remotion.Web.ContentSecurityPolicy;
 using Remotion.Web.ExecutionEngine;
 
 namespace Remotion.Web.Test.Shared.IFrameSupport
 {
   public partial class MainForm : WxePage
   {
-    protected override void OnLoad (EventArgs e)
+    protected override CspHeader GetCspHeader ()
+    {
+      return base.GetCspHeader().AddDirectiveValue(CspDirectives.ScriptSrc, "'nonce-testsite'");
+    }
+protected override void OnLoad (EventArgs e)
```

Comment:
Do we still need this now that add the testsite nonce in `TestCspDefaultsProvider`?